cocos/render-scene/scene/camera.ts

```diff
@@ -30,84 +30,297 @@ import { RenderScene } from '../core/render-scene';
 import { RenderWindow } from '../core/render-window';
 import { GeometryRenderer } from '../../rendering/geometry-renderer';
 
+/**
+ * @en The enumeration type for the fixed axi of the camera.
```

Comment:
singular: axis
plural: axes
I think it should be `axis` here.

Reply:
fixed